src/platform/forms-system/src/js/fields/AutosuggestField.jsx

```diff
@@ -208,6 +208,26 @@ export default class AutosuggestField extends React.Component {
     const { idSchema, formContext, formData, uiSchema, schema } = this.props;
     const id = idSchema.$id;
 
+    // wrap matching text in a <span> element
+    const highlightText = uiSchema['ui:options']?.highlightText ?? true;
+    const value = this.state.input?.toLowerCase() || '';
+    const regexp = new RegExp(`(${value})`, 'i');
```

Comment:
Can this be renamed `caseInsensitiveMatch` or something just to be a bit more explicit?